### PR TITLE
more descriptive sndfile errors

### LINKED ISSUE #270: opaque errors
It would be awesome if the errors here included the filename. I don't see another way to figure out which sound file in my dataset is bad.
https://github.com/facebookresearch/wav2letter/blob/ff3f2351e43720d356f16c00bb99e1a13b85ac2a/src/feature/Sound.cpp#L104-L110

(Also cool would be the ability to say "I know my dataset is flawed, please just skip files that have critical errors")

### PR DESCRIPTION
**Pull request by lunixbochs**
This adds the filename to all sndfile errors and clarifies the messages a bit. Fixes #270

### CODE REVIEW

File: src/feature/Sound.cpp [hunk 1]
```diff
@@ -157,11 +158,13 @@ extern void saveSound(
     nframe = sf_writef_short(
         file, reinterpret_cast<short*>(input.data()), info.frames);
   } else {
-    LOG(FATAL) << "sndfile: unsupported write format";
+    LOG(FATAL) << "sndfile: unsupported write format - "
+               << filename;
```
Comment (jacobkahn) [inline src/feature/Sound.cpp]: Can you `clang-format` everything? This should be on the previous line

File: src/feature/Sound.cpp [hunk 2]
```diff
@@ -157,11 +158,13 @@ extern void saveSound(
...
   }
   sf_close(file);
   if (nframe != info.frames) {
-    LOG(FATAL) << "sndfile: write error";
+    LOG(FATAL) << "sndfile: write error - "
+               << filename;
```
Comment (jacobkahn) [inline src/feature/Sound.cpp]: ditto

**Comment by lunixbochs** [comment]
Where's your clang-format config? I don't see one in this repo, and I tried a few different ones and none of them match your current style.

**Comment by jacobkahn** [comment]
@lunixbochs — https://github.com/facebookresearch/wav2letter/commit/07025afbccb110f7c1e0cdc18583f21231cc49b9 adds a clang-format to flashlight and wav2letter - can you rebase on top of that and update the PR? Thanks.

**Comment by lunixbochs** [comment]
Done

**Pull request closed without merging**